features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -150,6 +150,12 @@
 #ifdef MBED_CONF_LWIP_PBUF_POOL_BUFSIZE
 #undef PBUF_POOL_BUFSIZE
```

Comment:
The undef wants to come out of the ifdef, as you're now explicitly setting it in all cases.

Reply:
A fix provided. I think I grasped what you were looking for here. Each driver needs to be able to set the PBUF_POOL_BUFSIZE explicitly. In the same manner as PBUF_POOL_SIZE is set.

Reply:
Ah, that wasn't actually what I meant, but you're right, that's consistent.